.replit

```diff
@@ -0,0 +1,2 @@
+language = "java10"
+run = "RUN"
```

Comment:
"RUN" command wont start the game.
try:
"cd THE\ GAME/;java Runner;cd .."

OR
"java Runner" (but for this command, move the .repl file in THE GAME folder)